rpm/xcache.spec

```diff
@@ -86,6 +86,7 @@ Summary: The ATLAS data federation cache server
 
 Requires: %{name} = %{version}
 Requires: wget
+Requires: xrootd-rucioN2N-for-Xcache
```

Comment:
@brianhlin can you review if this make sense in the sense of [SOFTWARE-3784](https://opensciencegrid.atlassian.net/browse/SOFTWARE-3784)

Reply:
Makes sense to me, the we'll have to tell users to `--enablerepo=osg-contrib` since that's where we currently ship the plugin